app/Console/Commands/InitCommand.php

```diff
@@ -228,7 +229,19 @@ private function maybeSeedDatabase(): void
 
     private function maybeSetUpDatabase(): void
     {
+        $attemptCount = 0;
+
         while (true) {
+            // In non-interactive mode, we must not endlessly attempt to connect.
+            // Doing so will just end up with a huge amount of "failed to connect" logs.
+            // We do retry a little, though, just in case there's some kind of temporary failure.
+            if ($this->inNoInteractionMode() && $attemptCount >= NON_INTERACTION_MAX_ATTEMPT_COUNT) {
```

Comment:
This surely will break? Shouldn't it be `self::NON_INTERACTION_MAX_ATTEMPT_COUNT`?

Reply:
You're probably right - I haven't written much PHP in ~10 years. I thought I had tested it, but in retrospect I guess I may have been running interactively at the time. Sorry, I'll fix that.